app/src/main/java/com/koalatea/sedaily/feature/commentList/epoxy/ReplyEpoxyModelWithHolder.kt

```diff
@@ -1,9 +1,27 @@
 package com.koalatea.sedaily.feature.commentList.epoxy
 
+import android.graphics.Typeface
+import android.widget.Button
+import android.widget.ImageButton
+import android.widget.TextView
+import com.airbnb.epoxy.EpoxyAttribute
 import com.airbnb.epoxy.EpoxyModelClass
 import com.koalatea.sedaily.R
 
 @EpoxyModelClass(layout = R.layout.view_holder_reply)
-abstract class ReplyEpoxyModelWithHolder : BaseCommentEpoxyModelWithHolder<ReplyHolder>()
+abstract class ReplyEpoxyModelWithHolder : BaseCommentEpoxyModelWithHolder<ReplyHolder>() {
 
-class ReplyHolder : BaseCommentHolder()
\ No newline at end of file
+    @EpoxyAttribute lateinit var upVoteClickListener: () -> Unit
+    @EpoxyAttribute var textWeight: Int = Typeface.NORMAL
```

Comment:
Let us not do that (have styles outside of the XML) and use an outline and filled icons instead in one custom drawable file like in https://github.com/SoftwareEngineeringDaily/software-engineering-daily-android/blob/develop/app/src/main/res/drawable/play_pause_src.xml and https://github.com/SoftwareEngineeringDaily/software-engineering-daily-android/blob/develop/app/src/main/java/com/koalatea/sedaily/feature/player/BasePlayerActivity.kt#L68

Icons: 
* https://material.io/resources/icons/?icon=thumb_up&style=outline
* https://material.io/resources/icons/?icon=thumb_up&style=baseline